src/Dropdown/README.md

```diff
@@ -47,6 +47,31 @@ notes: |
 </Dropdown>
 ```
 
+### With IconButton
+
+You can use `Dropdown.Toggle` with `IconButton` component, but be aware that by default it renders an arrow which messes up the positioning of the `IconButton` element. Paragon offers a helper CSS class `pgn__dropdown-iconbutton` which removes that arrow in case you don't need it, you should pass it as a `bsPrefix` prop, so that it overrides base react-bootstrap's class which adds the arrow. Also note that all props you provide to `Dropdown.Toggle` will get passed down to the `IconButton`.
```

Comment:
Hmm, when the arrow is rendered with the `IconButton`, the positioning is a bit off per the screenshots below. I'm thinking instead of fixing that we just don't show the arrow altogether using an `IconButton` with the `as` prop. 

I'm not sure there's a use case where we'd want the default dropdown arrow to show when using an `IconButton` as the dropdown target, since the target itself is already an icon.

Could we add the `pgn__dropdown-toggle-iconbutton` class by default when the `IconButton` is passed to `as`?

![image](https://user-images.githubusercontent.com/2828721/146374912-018503f3-8a53-47f6-9caa-b57cecbf3071.png)
![image](https://user-images.githubusercontent.com/2828721/146374928-2ea6e9d3-8115-4cc3-a3ca-afccf15c093a.png)

Reply:
Agree, thank you! I doubt someone will need to use that arrow with an `IconBUtton`, removed it.